app/services/support_interface/notifications_export.rb

```diff
@@ -0,0 +1,50 @@
+module SupportInterface
+  class NotificationsExport
+    def data_for_export
+      courses = Course.where(recruitment_cycle_year: RecruitmentCycle.current_year, open_on_apply: true)
+      providers = Provider.includes(:provider_users, :application_choices).joins(:courses).merge(courses).group('providers.id')
+      providers.flat_map do |provider|
+        data_for_provider(provider, provider_users_with_make_decisions(provider), visible_applications(provider))
+      end
+    end
+
+  private
+
+    def data_for_provider(provider, users_with_make_decisions, applications)
+      {
+        'Provider Code' => provider.code,
+        'Provider' => provider.name,
+        'No. Applications Received' => applications.count,
+        'No. Applications awaiting decisions' => applications.count(&:awaiting_provider_decision?),
+        'No. applications receiving decisions' => receiving_decisions_count(applications) - rbd_count(applications),
+        "No. applications RBD'd" => rbd_count(applications),
+        'No. provider users' => provider.provider_users.count,
+        'No. users with make_decisions' => users_with_make_decisions.count,
+        'No. of users with make_decisions and notifications disabled' => users_with_make_decisions.count(&:send_notifications),
```

Comment:
Should this be the opposite with the below as it's notifications disabled 🤔 ?

Reply:
good spot, flipped them around